### PR TITLE
docs(zone): expose zone

### PR DESCRIPTION
**Pull request by zmstone**
Fixes [EMQX-12162](https://emqx.atlassian.net/browse/EMQX-12162)

At the moment (5.7), we still do not have APIs to create a zone, or manage zone configs.
This PR only exposes the already in-place zone config by providing descriptions and examples.

### CODE REVIEW

File: en_US/configuration/configuration.md [hunk 1]
```diff
@@ -253,6 +251,55 @@ authentication = [{ enable = true }]
...
+    }
+    session_persistence {
+      # only apply durable storage for sessions in this zone
+      ...
+    }
+  }
+  my_zone2 {
+    ...
+  }
+}
```
Comment (wivwiv) [inline en_US/configuration/configuration.md:295]: Add a configuration that shows how to bind a listener to a created zone.

**Review by wivwiv — DISMISSED** [review]

File: zh_CN/configuration/configuration.md [hunk 2]
```diff
@@ -244,6 +244,52 @@ authentication = [{ enable = true }]
 
 :::
 
+### 区域覆盖
```
Comment (zmstone) [inline zh_CN/configuration/configuration.md]:
```suggestion
### Zone 覆盖 
```

File: zh_CN/configuration/configuration.md [hunk 3]
```diff
@@ -244,6 +244,52 @@ authentication = [{ enable = true }]
 
 :::
 
+### 区域覆盖 
+
+EMQX 中的区域是一种配置分组的概念。可以通过将 `zone` 字段设置为所需区域的名称，将区域与监听器关联。与某个区域关联的监听器连接的 MQTT 客户端将继承该区域的配置，这些配置可能会覆盖全局设置。
```
Comment (zmstone) [inline zh_CN/configuration/configuration.md]:
```suggestion
EMQX 中的区域是一种配置分组的概念。可以通过将监听器的 `zone` 字段设置为所需区域的名称，将区域与监听器关联。与某个区域关联的监听器连接的 MQTT 客户端将继承该区域的配置，这些配置可能会覆盖全局设置。
```

File: en_US/configuration/configuration.md [hunk 4]
```diff
@@ -253,22 +251,70 @@ authentication = [{ enable = true }]
...
+      # Enable durable storage for sessions in this zone
+      ...
+    }
+  }
+  my_zone2 {
+    ...
+  }
+}
+```
+
```
Comment (wivwiv) [inline en_US/configuration/configuration.md:297]:
```suggestion
In any listener, set the zone field to associate with a zone that has been created.

```bash
listeners.tcp.default {
   bind = 1883
   zone = my_zone1
   ...
}
```

```

**Review by wivwiv — CHANGES_REQUESTED** [review]

**Pull request merged**